File: src/back/BLL/Dtos/HelpModels/QuestionWithAnswer.cs
namespace back.BLL.Dtos.HelpModels
{
    public class QuestionWithAnswer
    {
        public int QuestionId { get; set; }
        public int AnswerId { get; set; }
        public string QuestionText { get; set; }
        public string AnswerText { get; set; }
    }
}

File: src/Front/app/src/main/java/com/example/front/model/product/QuestionWithAnswer.kt
package com.example.front.model.product

import com.google.gson.annotations.SerializedName

data class QuestionWithAnswer(
    @SerializedName("QuestionId") val questionId: Int,
    @SerializedName("AnswerId") val answerId: Int,
    @SerializedName("QuestionText") val questionText: String,
    @SerializedName("AnswerText") val answerText: String
)
